cloud/amazon/ec2_ami.py

```diff
@@ -44,7 +44,7 @@
     default: 300
   state:
     description:
-      - create or deregister/delete image
+      - create (present), register (register), deregister/delete image (absent)
```

Comment:
that was my whole point in 'put them both under present', detect which one is being used and deal with the diff actions underneath, the user doesn't really need diff states, just diff inputs.